packages/flutter_plugin_android_lifecycle/android/src/test/java/io/flutter/embedding/engine/plugins/lifecycle/FlutterLifecycleAdapterTest.java

```diff
@@ -95,5 +95,11 @@ public void addOnSaveStateListener(
     @Override
     public void removeOnSaveStateListener(
         @NonNull ActivityPluginBinding.OnSaveInstanceStateListener listener) {}
+
+    public void addOnWindowFocusChangedListener(
```

Comment:
I'm curious if this will pass on master. I assumed the answer to the comment of @stuartmorgan was to suppress the warning about the unneeded `@Override`, but I assume that this isn't actually one of the lints that we run against.